src/schema/common.cljs

```diff
@@ -8,7 +8,20 @@
 (def ^:private ?customization-color
   [:or :string :keyword])
 
+(def ^:private ?public-key
+  [:re #"^0x04[0-9a-f]{128}$"])
+
+(def ^:private ?general-rpc-call
+  [:cat
+   [:map
+    [:method :string]
+    [:params [:sequential :any]]
+    [:on-success fn?]
```

Comment:
@yqrashawn, maybe you forgot to push changes? I see all threads are resolved, but I don't see code changes.

Reply:
yes, didn't push changes
I upgraded macos to 14 so that I can run debug build on ios 17 device
had some trouble running debug build in simulator so hadn't test my changes

just pushed the commit